File: flutter_secure_storage_x/android/src/main/java/com/it_nomads/fluttersecurestorage/ciphers/deprecated/RSACipher18Implementation.java
package com.it_nomads.fluttersecurestorage.ciphers.deprecated;

import com.it_nomads.fluttersecurestorage.ciphers.KeyCipher;

import java.security.Key;

public class RSACipher18Implementation implements KeyCipher {
    public RSACipher18Implementation() {
    }

    @Override
    public byte[] wrap(Key key) {
        throw new UnsupportedOperationException("Not implemented");
    }

    @Override
    public Key unwrap(byte[] wrappedKey, String algorithm) {
        throw new UnsupportedOperationException("Not implemented");
    }
}

File: flutter_secure_storage_x/android/src/main/kotlin/com/it_nomads/fluttersecurestorage/ciphers/deprecated/RSACipher18Implementation.kt
package com.it_nomads.fluttersecurestorage.ciphers.deprecated

import com.it_nomads.fluttersecurestorage.ciphers.KeyCipher
import java.security.Key

internal class RSACipher18Implementation : KeyCipher {
  override fun wrap(key: Key): Nothing {
    throw UnsupportedOperationException("Not implemented")
  }

  override fun unwrap(wrappedKey: ByteArray, algorithm: String): Nothing {
    throw UnsupportedOperationException("Not implemented")
  }
}
